docs/tfengine/schemas/resources.md

```diff
@@ -186,6 +186,11 @@
 | healthcare_datasets.iam_members.role | IAM role to grant. | string | true | - | - |
 | healthcare_datasets.name | Name of healthcare dataset. | string | true | - | - |
 | iam_members | Map of IAM role to list of members to grant access to the role. | object | false | - | - |
+| kubernetes_service_accounts | Kubernetes service accounts. See <https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/service_account>. | array() | false | - | - |
+| kubernetes_service_accounts.google_service_account_email | Email of the google service account the kubernetes service account should use to authenticate with other resources. | - | true | - | - |
+| kubernetes_service_accounts.name | Name of the KSA. | string | true | - | - |
+| kubernetes_service_accounts.namespace | Namespace to where the KSA will be created. | string | true | - | - |
+| kubernetes_service_accounts.provider | Kubernetes provider. | string | true | - | - |
```

Comment:
Could you explain here what a "provider" is? The [main documentation](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/service_account) that you link doesn't talk about what this value should be (other than the fact that it's _about_ the kubernetes Terraform provider), and neither does https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/.

Reply:
Linked to the terraform documentation.